test/search/test_organized_index.cpp

```diff
@@ -344,7 +357,8 @@ TEST (PCL, Organized_Neighbor_Search_Pointcloud_Neighbours_Within_Radius_Search)
 TEST (PCL, Organized_Neighbor_Search_Pointcloud_Neighbours_Within_Radius_Search_Benchmark_Test)
```

Comment:
This "test" doesn't really belong here?

It seems to capture processing times, but don't print or use them / compare them?

Reply:
That is true, this could be converted into a proper benchmark in the future. I guess for now we can just leave it as is.